jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -396,6 +400,13 @@ public Object run() {
  */
 @CallerSensitive
 public static Class<?> forName(String className) throws ClassNotFoundException {
```

Comment:
Is this change required for `forName(Module module, String name)` as well?

Reply:
yes; updated PR.